statefun-e2e-tests/statefun-routable-kafka-e2e/src/test/resources/Dockerfile

```diff
@@ -13,7 +13,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM statefun
+FROM flink-statefun:2.1-SNAPSHOT
```

Comment:
Note:
these will be changed to `FROM flink-statefun:2.0-SNAPSHOT` when backporting these changes to the snapshot `release-2.0` branch.

On release candidates / releases, these will be e.g. `FROM flink-statefun:2.0.0`

Reply:
The updated `update_branch_version.sh` script takes care of updating these strings automatically.